docs/development/integration_tests.md

```diff
@@ -31,8 +31,9 @@ If the Control Cluster is a Gardener Shoot cluster then,
 
 If the Control Cluster is a Gardener SEED cluster, then the suite ideally employs the already existing `MachineClass` and Secrets. However,
 
-1. Define the variable `IS_CONTROL_SEED` in the `.env` file and set it to `true`. 
+1. Define the variable `IS_CONTROL_CLUSTER_SEED` in the `.env` file and set it to `true`. 
 `Warning:` Make sure to set the `CONTROL_NAMESPACE` variable to the shoot namespace where the control plane of the target resides.
+1. Please pass `TARGET_RESOURCE_GROUP` in the `.env` file and set it to the name of the target cluster. This is compulsory for Azure clusters.
```

Comment:
the doc is designed for opensource use of IT of MCM. So its not necessary to set this variable to the `target cluster name`, but it should be set to the name of resource group (an azure specific concept) where the resources of the cluster are made by MCM . Its only in gardener shoot cluster cases this variable is set to the cluster name..
kindly improve the statement keeping the above in mind.

Reply:
Done